.travis.yml

```diff
@@ -9,6 +9,7 @@ julia:
   - nightly
 env:
   - CONDA_JL_VERSION="2"
```

Comment:
Still mandatory?

Reply:
Turns out I was testing on py27 locally anyways, so I just dropped it now :p